developer-docs-site/docs/nodes/aptos-deployments.md

```diff
@@ -14,21 +14,23 @@ Make sure to see the row describing [**What not to do**](#what-not-to-do).
-|**Faucet EDIT**| m |Managed by Aptos Team. | lltn |Deployed, managed and upgraded by you.| Managed by Aptos Labs and the community.|
-|**Purpose of the network**| m |The devnet is built to experiment with new ideas, improve performance and enhance the user experience.|lltn | The local testnet is for your development purposes and runs on your local computer.| For executing the Aptos Incentivized Testnet programs for the community.|
-|**Network status**| m |Mostly live, with brief interruptions during regular updates. |lltn | On your local computer. | **Live only during Incentivized Testnet drives**. |
-|**Type of nodes** |m |Validators and fullnodes. |lltn | Validators and fullnodes. | Validators and fullnodes.|
-|**How to run a node**| m |N/A, run by Aptos Labs team. |lltn | See the [tutorial](local-testnet/using-cli-to-run-a-local-testnet.md). | See the node deployment guides (ADD-LINK).|
-|**How to connect to the network**|m |<ul><li> You can transact directly with devnet without a local testnet. See, for example, [Your first transaction](../tutorials/first-transaction.md).</li><li> You can run fullnodes locally on your computer and synchronize with devnet. See [Run a Fullnode](/nodes/full-node/public-fullnode).</li></ul>|lltn | You can start a Validator and fullnode locally and connect with your local testnet for development purposes. | You must start both a local AIT validator node locally to connect to the AIT. Optionally, fullnodes can also be run locally and connected to AIT.|
-|**TypeScript / JavaScript SDK to use EDIT**|m |The latest version of the [aptos](https://www.npmjs.com/package/aptos) package. The package on npmjs is tested and released with devnet. | lltn |Use the TS / JS SDK built from the same commit as the local testnet. See [this document](../guides/local-testnet-dev-flow) for an end-to-end guide explaining this flow. | N/A, do not develop against AIT. |
-|<span id="what-not-to-do">**What not to do**</span>| m |Do not attempt to sync your local AIT fullnode or AIT validator node with devnet. |lltn | Do not attempt to sync your local testnet validator node with AIT. | Make sure you deploy your local AIT fullnode, AIT validator node and AIT validator fullnode in the test mode, and follow the instructions in the node deployment guides (ADD LINK).|
+|Description | Mainnet | Devnet | Long-lived Testnet | Aptos Incentivized Testnet (AIT)|
+|---|---|---|---|---|
+|**REST API URL EDIT**| m |https://fullnode.devnet.aptoslabs.com/v1 | https://fullnode.testnet.aptoslabs.com/v1 | Available during AIT program. |
+|**Chain ID**| [Click here and **select Mainnet from top right**](https://explorer.aptoslabs.com/?network=mainnet). |[Click here and **select Devnet from top right**](https://explorer.aptoslabs.com/?network=Devnet).| [Click here and **select Testnet from top right**](https://explorer.aptoslabs.com/?network=Testnet).| Available during AIT program.|
```

Comment:
Looks like if we lowercase the network, the Explorer goes to the correct tab without the need to click anything:
https://explorer.aptoslabs.com/?network=testnet

And the Devnet link appears to work with no clicks either.

Reply:
Work in progress still. I want to wait for to confirm these links in the last minute when the mainnet is up.